src/core/Akka/Akka.csproj

```diff
@@ -53,7 +53,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
+    <DefineConstants>$(DefineConstants);CORECLR;AKKAIO</DefineConstants>
```

Comment:
Have you checked Akka.IO unit tests? Because right now all of them disabled (no AkkaIO constant in Akka.Tests.csproj)